lib/spack/spack/cmd/activate.py

```diff
@@ -34,9 +34,6 @@
 
 
 def setup_parser(subparser):
-    subparser.add_argument(
```

Comment:
You could set `with_dependencies=False` in the below call to `do_activate` if this was true (admittedly this was not being used before though).